src/h5web/providers/mock/utils.ts

```diff
@@ -1,15 +1,13 @@
-import mockData from './data.json';
-import type { HDF5SimpleShape } from '../models';
+import { mockMetadata } from './data';
+import { assertSimpleShape } from '../utils';
 
-const datasets = Object.values(mockData.datasets);
+export function getMockDatasetDims(name: string): number[] {
```

Comment:
Retrieving a dataset's value in the stories is now trivial - e.g. `mockValues.twoD`, so I removed `getMockedDataset`. However, I had to replace with `getMockDatasetDims` to help retrieve a dataset's dimensions based on its name.